ruby-v2/.gitignore

```diff
@@ -0,0 +1,2 @@
+Gemfile.lock
+.idea/
```

Comment:
I think this should be in your [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) instead of here 🙂